recipes/dwclib/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "dwclib" %}
+{% set version = "2022.3.22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41e5dc52511e2f3591866699796f0d43aad1ee37b176cae5034b4ec30c99d6ac
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - flit-core >=2,<4
+  host:
+    - python >=3.6
+    - pip
+    - flit-core >=2,<4
+  run:
+    - python >=3.6
+    - numba >=0.53.1
+    - sqlalchemy >=1.3
+    - pyodbc >=4.0.32
+    - psycopg2 >=2.9.3
+    - dask >=2021.0
+
+test:
+  imports:
+    - dwclib
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://framagit.org/jaj/dwclib
+  summary: 'Python wrapper to DataWarehouse Connect'
+  description: |
+    dwclib is a Python library which enables queries against Philips
+    DataWarehouse Connect databases in a simple and fast manner.
+    Numerics and Waveform data can be queried and is returned as pandas
+    dataframes. Optional dask support is included in the library.
+    Waveform queries are accelerated with the numba JIT compiler.
+  license: ISC
+  license_family: MIT
```

Comment:
This isn't an MIT license and these key is optional these days:
```suggestion
```

Reply:
Thank you very much for the review. I commited both changes and confirm it build correctly under docker.